roman_numeral_converter_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "./roman_numeral_converter.rb"
+
+describe "Roman Numeral Converter" do
+  context "roman numeral to arabic conversion" do
+    before { @converter = RomanToArabicConverter.new }
+
+    context "roman numeral is valid" do
+      context "roman numeral is one character" do
+        specify { expect(@converter.convert("I")).to eq(1) }
+        specify { expect(@converter.convert("V")).to eq(5) }
+        specify { expect(@converter.convert("X")).to eq(10) }
+        specify { expect(@converter.convert("L")).to eq(50) }
+        specify { expect(@converter.convert("C")).to eq(100) }
+        specify { expect(@converter.convert("D")).to eq(500) }
+        specify { expect(@converter.convert("M")).to eq(1000) }
+      end
+      context "addition between roman numerals" do
+        specify { expect(@converter.convert("CXI")).to eq(111) }
+        specify { expect(@converter.convert("CCC")).to eq(300) }
+        specify { expect(@converter.convert("DIX")).to eq(509) }
+      end
+      context "roman numerals that are two character long" do
+        specify { expect(@converter.convert("IV")).to eq(4) }
+        specify { expect(@converter.convert("IX")).to eq(9) }
+        specify { expect(@converter.convert("XL")).to eq(40) }
+        specify { expect(@converter.convert("XC")).to eq(90) }
+        specify { expect(@converter.convert("CD")).to eq(400) }
+        specify { expect(@converter.convert("CM")).to eq(900) }
+      end
+    end
```

Comment:
:question: Would it be worthwhile to add a few more tests that show a bit more expansive examples? 
__E.G.__ _MCMXLIV -> 1944_